Packs/MicroFocusSMAX/Integrations/MicroFocusSMAX/MicroFocusSMAX.yml

```diff
@@ -0,0 +1,545 @@
+category: Case Management
+commonfields:
+  id: MicroFocusSMAX
+  version: -1
+configuration:
+- defaultvalue: Incident
+  display: Object To Fetch
+  hidden: false
+  name: object_to_fetch
+  options:
+  - Incident
+  - Request
+  required: false
+  type: 15
+- defaultvalue: https://us1-smax.saas.microfocus.com
```

Comment:
Hey @ayman-m, it seems like you are failing on the secrets validations.
You can either remove the secret or add it to the secrets-ignore file.
For more information read here https://xsoar.pan.dev/docs/concepts/demisto-sdk#secrets

Reply:
Hey @iyeshaya , I think this is just a false positive ,  https://us1-smax.saas.microfocus.com is not a secret, it's Microfocus cloud console url

Reply:
That's exactly why we have the secret-ignore file, everything inside this file will be ignored during the secret validation process.
Please add ```https://us1-smax.saas.microfocus.com``` to this file.
Thank you.

Reply:
I have added it to .secrets-ignore